src/witan/validation.clj

```diff
@@ -12,21 +12,37 @@
   (let [ext (last (clojure.string/split filename #"\."))]
     (= ext "csv")))
 
-(defmulti category-valid?
+(defmulti category-validation-error?
   (fn [category lines] category))
 
-(defmethod category-valid? development-data-category
+(defn all-years-starting-from-2011
+  [year-headers]
+  (let [years (sort (map #(Long/parseLong (re-find #"\d{4}" %)) year-headers))]
+    (=  (vec (range 2011 (inc (last years))))
+        years)))
+
+(defmethod category-validation-error? development-data-category
```

Comment:
was a multimethod because I foresee having documents for several categories with different validations - though at this point you're right that we don't need one since we only validate development-data type documents.